lib/helpers/sfConnection.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const common = require('../common.js');
+const jsforce = require('jsforce');
+const debug = require('debug')('salesforce:auth');
+
+
+exports.createConnection = function (configuration, emitter) {
+  const connection = new jsforce.Connection({
+    oauth2: {
+      clientId: process.env.SALESFORCE_KEY,
+      clientSecret: process.env.SALESFORCE_SECRET,
+      redirectUri: 'https://app.elastic.io/oauth/v2',
```

Comment:
Redirect URI should be dynamically generated based on domain of an installation.
Please, replace this hardcoded URI with dynamic one.
`redirectUri: 'https://app.elastic.io/oauth/v2'`